### PR TITLE
Item admin mange user account mary

### PR DESCRIPTION
**Pull request by picka18**
Create Layout 
Admin generate Report
Admin Manage User Account

3-12-2024
9:43pm

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="green">#1D7844</color>
-    <color name="lightgreen">#D5F1E1</color>
-</resources>
+<!--    <color name="black">#FF000000</color>-->
```
Comment (RevyaS) [inline app/src/main/res/values/colors.xml:3]: Delete comments

File: app/src/main/res/layout/fragment_admin_manage_user_account.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        android:id="@+id/textViewTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Manage Admin Accounts"
+        android:textSize="24sp"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="16dp"/>
+
+    <!-- List of Admin Accounts -->
+    <ListView
```
Comment (RevyaS) [inline app/src/main/res/layout/fragment_admin_manage_user_account.xml:18]: Please use recycler view for listing admin accounts

Comment (picka18) [reply app/src/main/res/layout/fragment_admin_manage_user_account.xml:18]: done change recycler view and cards

**Comment by RevyaS** [comment]
Approved

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (picka18) [reply app/src/main/res/values/colors.xml]: done delete comments

**Pull request merged**